pkg/env/roxctl.go

```diff
@@ -46,4 +46,10 @@ var (
 
 	// ScannerDBDownloadBaseURL specifies the base URL to use when downloading offline scanner definitions.
 	ScannerDBDownloadBaseURL = RegisterSetting("ROX_SCAN_DB_DL_BASE_URL", WithDefault("https://install.stackrox.io/scanner"))
+
+	// OutputFile specifies the path where roxctl should write its standard output
+	OutputFile = RegisterSetting("ROX_OUTPUT_FILE")
```

Comment:
nit: maybe also just call it `ROX_STDOUT/ROX_STDERR`, the file suffix can be dropped IMO.